src/routes.tsx

```diff
@@ -229,6 +240,7 @@ export const reactPageRoutes: ReactPageRoute[] = [
                   baseUrl: '/se/new-member/switch',
                   name: 'Web Onboarding - Swedish Switcher',
                 }
+              // TODO: Remove this or? 👇
```

Comment:
Hm, yeah I guess this isn't used :D

Reply:
OK let's remove it later then since I really want to merge this now 😄